docs/AirGapImageConfiguration.md

```diff
@@ -56,6 +56,7 @@ spec:
 | Results               | api                               | `IMAGE_RESULTS_API`                                 |
 | Results               | postgres                          | `IMAGE_RESULTS_POSTGRES`                            |
 | Results               | watcher                           | `IMAGE_RESULTS_WATCHER`                             |
+| Results               | retention-policy-agent            | `IMAGE_RESULTS_RETENTION_POLICY_AGENT`              |
```

Comment:
@pratap0007 please move it to openshift section, I believe it is not available for kubernetes